react/features/base/connection/reducer.js

```diff
@@ -37,6 +38,12 @@ ReducerRegistry.register(
 
         case SET_ROOM:
             return _setRoom(state);
+
+        case SHOW_CONNECTION_INFO:
+            return {
```

Comment:
There should be a check here in case the new action is the same as the old one so we don't trigger a useless re-render.